modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -174,28 +248,45 @@ definition {
 				thirdInvoice = "Document_1.png");
 		}
 
-		task ("And I go back to the MDF listing page") {
-			PRMUtils.gotoPRM();
+		task ("And given that the Manager user is logged in and goes to MDF listing page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
 
-			PRMHomePage.gotoQuickLinksOption(
-				buttonLink = "mdf-requests",
-				buttonText = "MDF");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
 		}
 
 		task ("And I go to MDF claim listing page") {
 			PRMNavigator.goToNavItem(navItem = "MDF Claim");
 		}
```

Comment:
Hi @marinaasant0s, this step isn't necessary. You can use `PRMNavigator.openSitePage(pageName = "MDF Claim");`